portfolio/src/main/webapp/style.css

```diff
@@ -1,4 +1,63 @@
+#main-nav {
+  background-color: #212529;
+  padding-bottom: 0.7rem;
+  padding-top: 0.7rem;
+}
+#main-nav .navbar-toggler {
+  background-color: #fed136;
+  border: 0;
+  font-family: "Montserrat", -apple-system, BlinkMacSystemFont, Arial, sans-serif;
+  font-size: 0.75rem;
+  padding: 0.75rem;
+  text-transform: uppercase;
+}
+#main-nav .navbar-brand {
+  color: #ffc107;
+  font-family: "Montserrat", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif;
```

Comment:
Why add more alternative fonts here than in the other font-family declarations?

Reply:
I sketched the navbar style on codepen.io to avoid any potential bug. The starter navbar provided me many more fonts. I usually chose one, append -apple-system, Arial, sans-serif and delete the rest.
I modified right now to maintain code consistency.